.github/workflows/pr-test.yml

```diff
@@ -66,7 +66,7 @@ jobs:
      
       - name: Publish Unit Test Results
         uses: docker://ghcr.io/enricomi/publish-unit-test-result-action:v1.6
-        if: always()
+        if: github.event_name == 'push'
```

Comment:
can we only exclude non-member pull requests? This will exclude member's PR too?

Reply:
@kotharironak this is for pushing the test results, not for actually displaying the coverage report isn't @JBAhire ?

Reply:
@kotharironak this is more of a permissions issue. There's no out of the box way to check it `github.actor` is org member as of now. You can set condition to repository owner but that doesn't solve the problem. There are hacks like [this](https://github.community/t/when-github-actor-is-a-member-not-github-repo-owner/153431/2) which are not worth here considering how small the issue is. 

And @jcchavezs , this is for displaying comment with test summary like this: https://github.com/hypertrace/document-store/pull/35#issuecomment-779141554 . 

This use-case is also satisfied by codecov upto some extent now so I won't bother enough about it as of now.

Reply:
I was referring to `github.event.pull_request.head.repo.full_name != github.repository` to check if this is forked PR

Reply:
yup. updated